crates/winmd/src/types/delegate.rs

```diff
@@ -65,10 +65,10 @@ impl Delegate {
             unsafe impl<#constraints> ::winrt::RuntimeType for #name {
                 type Abi = ::winrt::RawPtr;
                 fn abi(&self) -> Self::Abi {
-                    self.ptr.get()
+                     <::winrt::IUnknown as ::winrt::ComInterface>::as_vtable(&self.ptr) as Self::Abi
```

Comment:
This seems a lot more complicated... 😉

Reply:
To be fair this is just because we want to ensure no ambiguity in code gen. If it were regular code it would just be `self.ptr.as_vtable()`

Reply:
Oh right, got it.